app/controllers/api/internal/posts_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Api::Internal::PostsController < Api::Internal::BaseController
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Not covered lines: 1 to 34

Reply:
El mono está mareado, porque simple cov me dice que tengo el 💯 %
![Captura de Pantalla 2022-05-13 a la(s) 10 11 47](https://user-images.githubusercontent.com/30674444/168302407-1ee09d1f-3582-44b9-aa71-b583e55062a3.png)